packages/application/src/services.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// A list of supported docker images for submission
+// TODO: replace this static list with a call to a metadata service
+const dockerImages: { [key: string]: string } = {
+  'tensorflow/tensorflow:1.12.3-py3': 'Tensorflow 1.12.3',
+  'tensorflow/tensorflow:1.12.3-gpu-py3': 'Tensorflow 1.12.3 w/ GPU',
+  'tensorflow/tensorflow:2.0.0-py3': 'Tensorflow 2.0',
+  'tensorflow/tensorflow:2.0.0-gpu-py3': 'Tensorflow 2.0 w/ GPU',
+  'pytorch/pytorch:1.2-cuda10.0-cudnn7-runtime': 'Pytorch 1.2 w/ CUDA-runtime',
+  'pytorch/pytorch:1.2-cuda10.0-cudnn7-devel': 'Pytorch 1.2 w/ CUDA-devel',
+  'amancevice/pandas:0.25.0-jupyter': 'Pandas'
+};
+
+/**
+ * A utilities class for various elyra services.
+ */
+export class ElyraServices {
```

Comment:
I am over the fence if we should call this Elyraxxxx, maybe FrontendServices or ApplicationServices

Reply:
I have no feeling about the name, I picked it on a whim just to get something down. I think I had though I'd eventually put the service classes here, but later realized they would be better in their own classes/files.

We can pick a name real quick in scrum and I'll update this